content/derisking-government-tech/resources/kickoff.md

```diff
@@ -1,9 +1,180 @@
 ---
-title: Sample kick-off agenda
+title: Sample kick-off week agenda
 permalink: /derisking-government-tech/resources/kickoff/
 layout: layouts/page
 tags: derisking-government-tech
-sidenav: false
+description: Sample kick-off week agenda for starting work with a vendor development team building custom software for a government agency.
+eleventyNavigation:
+  parent: derisking-government-tech-resources-kickoff
+  key: kickoff
+  order: 1
+  title: Sample kick-off week agenda
+sidenav: true
+sticky_sidenav: true
+subnav:
+- text: Day 1
+  href: '#day-1'
+- text: Day 2
+  href: '#day-2'
+- text: Day 3
+  href: '#day-3'
+- text: Day 4
+  href: '#day-4'
+- text: Day 5
+  href: '#day-5'
 ---
 
-TK
+{% include 'derisking-government-tech/pdf_document_download.html' doc_type_name: "agenda" pdf_url: "/assets/derisking-government-tech/dist/18F-de-risking-guide-v2-sample-kick-off-week-agenda.pdf" %}
+
+---
+
+<table class="kick-off-table">
+  <caption><h3 id="day-1">Day 1</h3></caption>
```

Comment:
It looks like this page skips from an `h1` to `h3`s

Reply:
I've updated these to `h2`s.